tests/providers/docker/operators/test_docker.py

```diff
@@ -22,6 +22,7 @@
 
 import pytest
 from docker.errors import APIError
+from docker.constants import DEFAULT_TIMEOUT_SECONDS
```

Comment:
```suggestion
from docker.constants import DEFAULT_TIMEOUT_SECONDS
from docker.errors import APIError
```

to fix static checks